man/udevadm.xml

```diff
@@ -175,6 +175,13 @@
             <para>Cleanup the udev database.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><option>-w</option></term>
+          <term><option>--wait-for-initialization<optional>=SECONDS</optional></option></term>
+          <listitem>
+            <para>Wait for device to be initialized.</para>
```

Comment:
This says nothing about the default. I think this should say `"If argument <replaceable>seconds</replaceable> is not specified, the default is to wait forever."`.